src/AppBundle/Resources/views/Decklist/toolbar.html.twig

```diff
@@ -13,6 +13,7 @@
 	</button>
 	<ul class="dropdown-menu" role="menu">
 		<li><a href="#" id="btn-download-text">{{ 'decks.form.download.text' | trans }}</a></li>
+		<li><a href="#" id="btn-download-text-cycle-ordered">{{ 'decks.form.download.text_cycleorder' | trans }}</a></li>
 		<li><a href="#" id="btn-download-octgn">{{ 'decks.form.download.octgn' | trans }}</a></li>
```

Comment:
for consistency reasons, add "order by release date, deluxe after" option here as well.

```html
<li><a href="#" id="btn-download-text-cycle-ordered-after">{{ 'decks.form.download.text_cycleorder_deluxe_after' | trans }}</a></li>
```

also, these buttons need to be wired up in `ui.decklist.js`. in the `do_action_decklist()` event handler, rig up routing for the new download options. 

something like this:

```js
            case 'btn-download-text-cycle-ordered':
                location.href = Routing.generate('decklist_export_text_ordered_deluxe_before', {decklist_id: app.deck.get_id()});
                break;
            case 'btn-download-text-cycle-ordered-after':
                location.href = Routing.generate('decklist_export_text_ordered_deluxe_after', {decklist_id: app.deck.get_id()});
                break;
```

Reply:
Solved in 6e6c54df53086d6174d428bc51feab923d66d56f (also, thanks for the explanation :smile:)